lib/app/modules/settings/views/settings_view.dart

```diff
@@ -17,6 +18,9 @@ class SettingsView extends GetView<SettingsController> {
 
   HomeController homeController = Get.find<HomeController>();
   ThemeController themeController = Get.find<ThemeController>();
+  SettingsController settingsController=Get.put(SettingsController());
```

Comment:
I think you should update the bindings rather than the view file to fetch settingsController.